ale_linters/pony/ponyc.vim

```diff
@@ -0,0 +1,21 @@
+" Description: ponyc linter for pony files
+
+call ale#Set('pony_ponyc_executable', 'ponyc')
+call ale#Set('pony_ponyc_options', '--pass paint')
+
+function! ale_linters#pony#ponyc#GetExecutable(buffer) abort
+    return ale#Var(a:buffer, 'pony_ponyc_executable')
+endfunction
+
+function! ale_linters#pony#ponyc#GetCommand(buffer) abort
+    return ale_linters#pony#ponyc#GetExecutable(a:buffer)
```

Comment:
That will need to be escaped in case the path contains spaces or something. I'll sort that out.